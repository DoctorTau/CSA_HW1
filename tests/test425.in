741
-2615 -2038 4896 206 -926 -1660 -518 2357 1204 5478 9308 -7824 7456 -9442 -4213 9035 3331 8205 -7461 -5856 7147 3519 -9457 2548 3283 3979 2226 -3449 -6342 -413 8902 4848 8622 8509 2053 -1686 3735 -6525 -1615 -312 575 -5658 -447 -7856 3307 -9785 2242 -3214 -4551 -6167 5624 -3266 -2226 2346 -6878 3684 -8449 3707 1030 5723 -6951 7172 2696 -8630 -9463 2379 -1941 -9102 92 -2271 -6836 4398 4823 -4289 3300 -4754 9535 4184 7817 -2429 7353 8688 -3947 235 9652 9882 -7674 4892 4957 -7467 -3663 -7087 -8380 -2365 7832 1550 -9103 -686 1702 -4836 -784 6735 9777 -48 -1946 -8642 1459 8740 -8868 -7970 7858 -4153 2167 5651 5781 3480 -287 -4451 -7014 -6348 2287 843 2028 9309 129 4043 -7434 -6365 -5952 5700 -1282 -3023 2475 2783 2929 -317 -7654 5950 -365 59 7679 7966 1618 9546 6427 7211 1844 2359 6785 -1020 5808 -5497 -2610 -5914 -8960 -7557 4403 2169 5470 8232 -8659 6678 -4483 9638 844 5382 6362 -9905 2346 500 -4863 2932 5983 -3880 -838 5207 -544 -3109 -6639 8358 -4055 -8119 -1611 5823 496 -5061 1787 7189 -5587 -7859 9282 3050 -3374 4655 9571 -910 -9165 -5483 9542 9699 -1804 7752 -7152 -8741 -5255 4405 2206 3214 8487 -7041 -8485 7006 -2375 -989 -5932 -1494 705 4868 4916 3220 -7807 6656 4514 -4688 -7922 8399 -6156 3738 -4557 -3449 -2318 9602 -6870 -1256 -1422 -5076 2339 -3635 659 564 6494 8122 7055 5600 8918 -9473 8984 2901 -977 2449 5999 -6131 -1043 -9994 9096 1275 3723 6038 2258 909 1980 -7065 8160 -8793 -943 8946 5425 150 -2045 -7608 1452 961 -2422 3382 -6928 5330 653 -3483 -3107 7111 3150 -7564 -3833 -207 -9744 -5987 -7085 -3401 8874 4146 9399 -5464 5235 -6751 -6474 -2705 9056 -270 -2141 9914 855 -1203 2832 -4842 9408 2355 3647 -3787 -925 -8913 9559 7353 119 -9161 7794 -5148 -5348 -264 9828 237 9597 -3862 -5600 -1750 -137 7066 -737 -7405 2416 -8779 743 -7129 459 -2356 6771 -5944 -5652 -4104 2979 -8653 -9149 -4453 3774 7433 1622 -344 -4931 3427 9090 -5614 -8353 7115 670 -7896 5836 4513 3584 -2311 -2598 1062 -4761 -3822 2528 -1707 7414 -4148 3729 -9654 -9093 7629 9395 8618 8122 6179 2747 -6096 -6937 3965 2859 5747 -244 -1138 9399 2764 -7522 1095 -5077 -3479 2859 -6666 996 2788 -3298 9962 7828 -2668 -7592 -4519 -2637 -1205 -5870 -3195 -9305 1939 -444 -98 2194 4516 875 -3481 -4846 7844 -2716 -969 -2995 9294 -8394 -6471 1372 5609 -2343 -911 9184 -102 -6400 1205 3949 7404 5528 -9277 -957 -6408 -4811 -7123 6505 1365 5595 -7459 -5701 -7757 2670 -6103 1347 3920 3548 4414 -3985 8315 -6391 3327 1843 8648 7900 -7793 4458 464 -6740 -3066 8095 9809 6967 738 -5437 -1211 8069 -8644 -6617 3638 -2372 4433 -3753 -3580 6615 5402 -4360 -1740 1280 4512 -8672 -8730 -8555 -7440 -8433 -6274 -4290 3876 -2688 -4137 1971 6217 -7735 -6338 5212 5835 -7869 3439 8447 -8542 9348 -3323 3536 8705 2101 6933 7457 9198 6078 9713 3148 -7378 3323 -4120 4972 1702 -564 5653 9734 -593 -2801 -5473 9225 -4110 -2717 7896 5424 -7376 -9951 3915 8187 9040 -4713 42 1224 -8606 -4045 -114 8225 -2338 -2971 7951 -8706 -2673 4572 5366 3717 -2135 9729 -4158 -8744 -5620 59 -9020 7082 5560 9037 5350 295 7110 -7647 -2407 -193 -8156 703 -2225 5374 1744 6917 333 7672 7584 -961 -5481 3262 953 -6532 -6425 1517 1885 -833 -5009 -84 8576 -570 6312 1097 -4231 4193 -3069 -5862 -2108 -8282 3721 -5994 -4438 -572 4276 2856 2583 -5719 -9140 -8144 135 3480 6777 -6211 2198 4782 -9604 -2185 6922 4152 8121 7212 5640 5084 8403 4601 3392 5649 8660 1474 2704 3139 7208 2201 911 1385 9420 -9008 7234 7336 2046 -5536 -9270 -9373 7676 5701 2233 5241 -6319 -497 -2500 7093 9174 397 9365 -3197 7071 -7045 9343 4930 795 -5350 5979 -7289 4671 2647 -7379 -5038 8241 5447 5441 5510 -9610 -8699 5273 2067 -7906 -2158 -1192 6895 7635 -691 -1256 -7961 8397 3534 -8226 -9518 1554 -8854 -7736 2651 3135 -7003 -4843 1766 -3137 6721 -3188 -1795 -3352 -7273 5504 1318 178 -4634 -60 6821 9961 -6204 2528 8935 -6669 236 -7023 318 -2681 -4104 -8709 7899 -7053 -1954 -4022 2537 -4852 2155 -8934 -2131 -6241 1678 7287 9153 -2023 5449 -3105 9510 2488 5048 2276 -2148 5738 -7883 -8822 -7211 -2688 7820 8555 -4277 8967 1896 