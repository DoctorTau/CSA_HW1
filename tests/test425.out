-2038 -926 -1660 -518 -7824 -9442 -5856 -6342 -1686 -312 -5658 -7856 -3214 -3266 -2226 -6878 -8630 -9102 -6836 -4754 -7674 -8380 -686 -4836 -784 -48 -1946 -8642 -8868 -7970 -7014 -6348 -7434 -5952 -1282 -7654 -1020 -2610 -5914 -8960 -3880 -838 -544 -3374 -910 -1804 -7152 -5932 -1494 -4688 -7922 -6156 -2318 -6870 -1256 -1422 -5076 -9994 -7608 -2422 -6928 -7564 -9744 -5464 -6474 -270 -4842 -5148 -5348 -264 -3862 -5600 -1750 -2356 -5944 -5652 -4104 -344 -5614 -7896 -2598 -3822 -4148 -9654 -6096 -244 -1138 -7522 -6666 -3298 -2668 -7592 -5870 -444 -98 -4846 -2716 -8394 -102 -6400 -6408 -6740 -3066 -8644 -2372 -3580 -4360 -1740 -8672 -8730 -7440 -6274 -4290 -2688 -6338 -8542 -7378 -4120 -564 -4110 -7376 -8606 -114 -2338 -8706 -4158 -8744 -5620 -9020 -8156 -6532 -84 -570 -5862 -2108 -8282 -5994 -4438 -572 -9140 -8144 -9604 -9008 -5536 -9270 -2500 -5350 -5038 -9610 -7906 -2158 -1192 -1256 -8226 -9518 -8854 -7736 -3188 -3352 -4634 -60 -6204 -4104 -1954 -4022 -4852 -8934 -2148 -8822 -2688 